graylog2-server/src/main/java/org/graylog2/rest/resources/system/indices/RotationStrategyResource.java

```diff
@@ -79,7 +80,7 @@ public RotationStrategies list() {
                 .stream()
                 .filter(this::isEnabledRotationStrategy)
                 .map(this::getRotationStrategyDescription)
-                .collect(Collectors.toSet());
+                .collect(Collectors.toCollection(LinkedHashSet::new));
```

Comment:
Nice fix! Could we change the data type to List everywhere, so we signalize that the collection is ordered?

Reply:
Yes, I can. I won't propagate this change outside of Resource/AutoValue classes, as I believe that in other places (like ElasticsearchConfiguration), it may be more important to signalize that we do not have duplicates in the collection. Does it sound ok?